pkgs/development/libraries/graphene/default.nix

```diff
@@ -70,17 +72,18 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    "-Dgtk_doc=${lib.boolToString (stdenv.buildPlatform == stdenv.hostPlatform)}"
-    "-Dintrospection=${if (stdenv.buildPlatform == stdenv.hostPlatform) then "enabled" else "disabled"}"
+    "-Dgtk_doc=true"
+    "-Dintrospection=enabled"
     "-Dinstalled_test_datadir=${placeholder "installedTests"}/share"
     "-Dinstalled_test_bindir=${placeholder "installedTests"}/libexec"
   ];
 
   doCheck = true;
 
   postPatch = ''
+    substituteInPlace tests/meson.build \
```

Comment:
Sounds very hacky :)

Reply:
wonder why i did that instead of just making the installedTests conditional on host == build